Sherlock/yoda/docu/docu_2_reference_code.cc

```diff
@@ -447,11 +452,14 @@ HTTP(port).ResetAllHandlers();
   // Primes that end with `7`.
   EXPECT_EQ("7,17,37,47,67,97",
             api.Transaction([](PrimesAPI::T_DATA data) {
-              // TODO(dkorolev): In real life, the order can be off. -- D.K.
               std::ostringstream os;
+              std::set<int> n;
```

Comment:
`n` bugs me -- it's commonly a counter. Perhaps `elements` or `values` or `entries`.